src/types.py

```diff
@@ -0,0 +1 @@
+ReplicaInfos = list[dict[str, object]]
```

Comment:
Maybe instead of `object` enumerate all possible values? There are not so much of them: int, str and that's all.